dusk/grammar.py

```diff
@@ -250,14 +250,52 @@ def statements(self, py_stmts: t.List, in_stencil_root_scope: bool = False):
         return sir_stmts
 
     @transform(
-        Assign(
-            targets=FixedList(Capture(expr).to("lhs")),
-            value=Capture(expr).to("rhs"),
-            type_comment=None,
+        OneOf(
+            Assign(
+                targets=FixedList(Capture(expr).to("lhs")),
```

Comment:
How does it work with the FixedList? Is it capturing the last element?

Reply:
`FixedList(x1, x2, x3, ..., xn)` will match `[x1, x2, x3, ..., xn]`. So technically it will match the last element in this case, but it will also be the only element and if the list doesn't have exactly one element, it will fail.